### ISSUE TITLE
"glances --browser" reports "Error while initializing the smart plugin ('NoneType' object has no attribute 'disable_smart')"

### ISSUE THREAD

**Issue by r4dh4l** [nicolargo/glances#1822]
**Describe the bug**

After installing glances from Debian 10.8 standard repositories on two of my VMs I wanted to access them with `glances --browser` following https://glances.readthedocs.io/en/latest/quickstart.html#central-client. I get a list of the expected VMs but without any data:

```
2 Glances servers available

  Name               LOAD     CPU%    MEM%    STATUS      IP                OS
> vm-seafile            ?        ?       ?       OFFLINE     192.168.1.166     ?
  vm-revproxy        ?        ?       ?       OFFLINE     192.168.1.126     ?
```

When I want to access one of the servers glances reports:

> Connect to vm-seafile Error while initializing the smart plugin ('NoneType' object has no attribute 'disable_smart')     

On both VMs I've opened port `61209`.

**To Reproduce**

1. Install glances on servers and client:

```
# apt install glances
# apt policy glances
glances:
  Installed: 3.1.0-1+deb10u1
  Candidate: 3.1.0-1+deb10u1
  Version table:
 *** 3.1.0-1+deb10u1 500
        500 http://ftp.de.debian.org/debian buster/main amd64 Packages
        100 /var/lib/dpkg/status
```
2. Open port `61209` on servers.
3. Configure `[serverlist]` section in `/etc/glances/glances.conf` on client.
4. Start `glances --browser` on client.

**Expected behavior**

I've expected that `glances --browser` would provide LOAD/CPU/MEM data similiar to https://glances.readthedocs.io/en/latest/quickstart.html#central-client with the described configuration.

**Desktop (please complete the following information):**
 - OS: `Debian 10.8`
 - Glances Version:
 
 ```
 $ glances -V
Glances v3.1.0 with psutil v5.5.1
 ```
 
 - Glances logs file: (sorry I couldn't find the log file location of glances on Debian)

**Comment by nicolargo** [comment]
Is every things ok when you click on one of your server ?

**Comment by nicolargo** [comment]
On my side the summary is displayed as expected

![Peek 2021-03-17 14-14](https://user-images.githubusercontent.com/776747/111473245-1de7b500-872b-11eb-98db-c00b22e8eea6.gif)

but... i have the following error when i click on my server:

```
python -m glances --browser
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nicolargo/dev/glances/glances/__main__.py", line 29, in <module>
    glances.main()
  File "glances/__init__.py", line 145, in main
    start(config=config, args=args)
  File "glances/__init__.py", line 114, in start
    mode.serve_forever()
  File "glances/client_browser.py", line 252, in serve_forever
    return self.__serve_forever()
  File "glances/client_browser.py", line 239, in __serve_forever
    self.__display_server(self.get_servers_list()[self.screen.active_server])
  File "glances/client_browser.py", line 189, in __display_server
    if not client.login():
  File "glances/client.py", line 162, in login
    if not self._login_glances():
  File "glances/client.py", line 130, in _login_glances
    self.stats.set_plugins(json.loads(self.client.getAllPlugins()))
  File "glances/stats_client.py", line 59, in set_plugins
    self._plugins[item] = plugin.Plugin(args=self.args)
  File "/home/nicolargo/dev/glances/glances/plugins/glances_network.py", line 65, in __init__
    self.hide_zero = config.get_bool_value(
AttributeError: 'NoneType' object has no attribute 'get_bool_value'
```

**Comment by nicolargo** [comment]
Try to disable the smart plugin on both client and server.

**Comment by r4dh4l** [comment]
Thanks for your reply @nicolargo 

> @r4dh4l Try to disable the smart plugin on both client and server.

Sorry, I couldn't find something "smart" related in the config:

```
root@j33-p-x86x64-nb-ed62b2c041:~# cat /etc/glances/glances.conf | grep smart
root@j33-p-x86x64-nb-ed62b2c041:~#
```

Where can I disable it?

**Comment by nicolargo** [comment]
glances --disable-plugin smart

**Comment by nicolargo** [comment]
Corrected in Glances 3.1.6 or higher.
